web/package.json

```diff
@@ -74,6 +74,7 @@
     "justified-layout": "^4.1.0",
     "lodash-es": "^4.17.21",
     "luxon": "^3.4.4",
+    "sharp": "^0.33.4",
```

Comment:
Why did you install sharp in the web?

Reply:
Anything, that is outside of the real source files, was not intentionally.


I saw, that this has changed, yes. But I assumed, that the setup steps have done that.
I just followed these steps.
https://github.com/immich-app/immich/blob/main/docs/docs/developer/setup.md#connect-web-to-a-remote-backend

Initially I had problems with the setup, Ubuntu LTS has way too old nodeJS and npm delivered in its package... took me some time to find that out...
See https://discord.com/channels/979116623879368755/1251343704975868005

After updating to the latest versions, all was fine.

vboxuser@ubuntults:~$ npm -v
10.7.0
vboxuser@ubuntults:~$ node -v
v20.14.0


Long story short, tell me, how I have done that, and I will revert it :)